### PR TITLE
Replaced exec with spawn to prevent buffer overflow

### PR DESCRIPTION
**Pull request by PikkamanV**
Related issue: #30

### CODE REVIEW

File: packages/cli/src/commands/start.ts [hunk 1]
```diff
@@ -65,7 +65,10 @@ const command: CommandModule = {
     if (typeof memcachedInfo === 'string') throw new Error('UNIX domain socket is unexpected')
 
     // run start command with env
-    const appProcess = exec(dappConfig.startAppCommand, {
+    const startAppCommandSplitted = dappConfig.startAppCommand.split(' ')
```
Comment (odanado) [inline packages/cli/src/commands/start.ts]: Change Request: use [shlex](https://www.npmjs.com/package/shlex) library to parse command line arguments.

File: packages/cli/src/commands/start.ts [hunk 2]
```diff
@@ -76,8 +79,12 @@ const command: CommandModule = {
         'EASY_MEMCACHED_PORT': memcachedInfo.port.toString()
       })
     })
-    appProcess.stdout.pipe(process.stdout)
-    appProcess.stderr.pipe(process.stderr)
+    appProcess.stdout.on('data', (data) => {
+      console.log(`${data}`)
```
Comment (odanado) [inline packages/cli/src/commands/start.ts]: Why you use template string?

File: packages/cli/src/commands/start.ts (hunk 1 above)
Comment (PikkamanV) [reply packages/cli/src/commands/start.ts]: Used shlex.

File: packages/cli/src/commands/start.ts (hunk 2 above)
Comment (PikkamanV) [reply packages/cli/src/commands/start.ts]: Used stdio option of spawn.

**Comment by odanado** [comment]
@PikkamanV run `npx lerna bootstrap --hoist` and commit lockfile

**Comment by PikkamanV** [comment]
@odanado updated.

**Pull request merged**